system/STM32L5xx/stm32l5xx_hal_conf_default.h

```diff
@@ -0,0 +1,497 @@
+/**
+  ******************************************************************************
+  * @file    stm32l5xx_hal_conf_default.h
+  * @brief   HAL default configuration file.
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2017 STMicroelectronics.
```

Comment:
This file is not correct as it does not match to the stm32l5xx_hal_conf_template.h

Reply:
The `system/STM32L5xx/stm32l5xx_hal_conf_default.h` 
is built like the 
`system/STM32L4xx/stm32l4xx_hal_conf_default.h`